### PR TITLE
Update script-topics.rpy movie and film

### PR DESCRIPTION
**Pull request by juank713**
so now wen you say the words movie or film she will talk about goin to the cinema and then sayng that she can't go, so she find a solution

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -38,7 +38,24 @@ init 11 python:
...
+    for key in ['film''movie']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_movie') #id
+
+label monika_movie:
+    m 1b "Do you want to watch a movie with me?!" 
+    m 1q"..."
+    m 1g "jeez, I can't go to the cinema with you."
+    m 1o"..."
+    m 1k "but I gues you can put the movie in the computer"
```
Comment (TacticalTriangle) [inline Monika After Story/game/script-topics.rpy]: Misspelled word.

Comment (TacticalTriangle) [reply Monika After Story/game/script-topics.rpy]: Also consider rewording this sentence sounds weird.

**Comment by Kiosho** [comment]
I'll be honest this topic needs some work, especially capitalization.
(Also, she will be able to watch a movie with you in the future. So this may become unnecessary...)

**Comment by juank713** [comment]
mmm... yea if she can watch movies in the future it will become use less

File: Monika After Story/game/script-topics.rpy (hunk 1 above)
Comment (juank713) [reply Monika After Story/game/script-topics.rpy]: ok, can you told me how to edit this, its my firs time so..
and my native lenguage is spanish, so i am not the best in english

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -38,7 +38,24 @@ init 11 python:
 
 #BEGIN ORIGINAL TOPICS
 
+
 #Use this topic as an example
+init 5 python:
+    # List of keywords for the topic.
+    for key in ['film''movie']:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: items in `python` lists are separated by commas ( , ). Please separate the strings in this list with a comma.

File: Monika After Story/game/script-topics.rpy (hunk 1 above)
Comment (juank713) [reply Monika After Story/game/script-topics.rpy]: nevermind

**Comment by juank713** [comment]
can any body help me

**Comment by monikLover** [comment]
Rejected for lack of content. Closing.

**Comment by juank713** [comment]
ok it was bad and deserve to get close

**Pull request closed without merging**